hera_corr_cm/hera_corr_cm.py

```diff
@@ -320,77 +345,115 @@ def get_ant_status(self):
             fem_pressure (float) : in mb
 
             Unknown values return the string "None"
+
+        Returns
+        -------
+        dict
+            keys are the parameter names to be stored, values are the parameter values
```

Comment:
I'd move the description of the returned dict under here. What is the second sentence in this docstring talking about? Is it about the output dict or is it about the stuff in redis?